apiml-security-common/src/main/java/org/zowe/apiml/security/common/filter/CategorizeCertsFilter.java

```diff
@@ -66,21 +66,21 @@ public class CategorizeCertsFilter extends OncePerRequestFilter {
      * @param request Request to filter certificates
      */
     private void categorizeCerts(ServletRequest request) {
-        X509Certificate[] certs = (X509Certificate[]) request.getAttribute(ATTRNAME_JAKARTA_SERVLET_REQUEST_X509_CERTIFICATE);
+        X509Certificate[] certs = (X509Certificate[]) request.getAttribute(ATTR_NAME_JAKARTA_SERVLET_REQUEST_X509_CERTIFICATE);
         if (certs != null && certs.length > 0 && certs[0] != null) {
```

Comment:
```suggestion
        if (certs != null && certs.length > 0) {
```
Why is there a check for the first element? `null` in there means an invalid structure.

Reply:
correct, with the new change it doesn't make sense to check for first element whether is null